src/pixmaps.c

```diff
@@ -3015,7 +3015,7 @@ GwHierarchyIcons *gw_hierarchy_icons_new(void)
     self->fork = gdk_pixbuf_new_from_xpm_data(arrow_divide);
 
     /* SV */
-    self->interface = gdk_pixbuf_new_from_xpm_data(arrow_inout);
+    self->iface = gdk_pixbuf_new_from_xpm_data(arrow_inout);
```

Comment:
Why did you decide to rename this?

Reply:
I couldn't get it to compile without it. There seems to be a `interface` define somewhere. I didn't investigate it.